pages/snapshots/[id].js

```diff
@@ -36,9 +36,19 @@ const SnapshotSummary = ({ initialSnapshot, token }) => {
   };
 
   const { dob, firstName, lastName, assets, vulnerabilities, notes } = snapshot;
+  const customerId = snapshot.systemIds?.[0];
 
   return (
     <>
+      <div>
+        <a
+          href={`${process.env.NEXT_PUBLIC_SINGLEVIEW_URL}/customers/${customerId}/view`}
+          className="govuk-back-link back-button"
+          data-testid="back-link-test"
+        >
+          Back to Singleview
```

Comment:
Just a thought, do we usually write it as "Singleview" or "Single View" elsewhere? Should make sure we are consistent across all the apps.